code/espurna/ntp.ino

```diff
@@ -268,13 +347,24 @@ void ntpSetup() {
             .onKeyCheck(_ntpWebSocketOnKeyCheck);
     #endif
 
-    // Main callbacks
-    espurnaRegisterLoop(_ntpLoop);
-    espurnaRegisterReload([]() { _ntp_configure = true; });
+    #if TERMINAL_SUPPORT
+        terminalRegisterCommand(F("NTP"), [](Embedis* e) {
+            _ntpReport();
+            terminalOK();
+        });
+
+        terminalRegisterCommand(F("NTP.SETTIME"), [](Embedis* e) {
+            if (e->argc != 2) return;
+            _ntp_synced = true;
+            _ntpSetTimestamp(String(e->argv[1]).toInt());
+            terminalOK();
+        });
 
-    // Sets up NTP instance, installs ours sync provider
-    _ntpStart();
+        // TODO:
+        // terminalRegisterCommand(F("NTP.SYNC"), [](Embedis* e) { ... }
+        //
```

Comment:
We can't exactly speed-up sntp like NtpClientLib, so idk about this.
Changing `ntpServer` value also triggers sync